### PR TITLE
Added headroom menu

### PR DESCRIPTION
**Pull request by asharonbaltazar**
Issue #281, implemented a headroom menu with a lightweight npm package (svelte-headroom). The header obviously broke some styling, which I did my best to fix. I also implemented an event to close the menu (if it's open) when scrolling. Please have a look. 

Edit : noticed more styling tweaks needed. Happy to discuss an orderly approach rather than haphazardly scattering `z-10`s :stuck_out_tongue_winking_eye:

**Next move:**
Based on this restyling, the next issue to focus on is adding a small redesign to the menu, particularly the padding. I believe the whitespace at the top of the page balanced it out. Have a look here:
image

### CODE REVIEW

**Comment by csell5** [comment]
At first glance, we need to rework that top nav's overall layout. As it's not taking up the whole width of the page.

image

You already mentioned the padding. I'd just make it equal on both the top and bottom so the logo is centered vertically.

And I'd love to jump on and chat more if you'd like. I'd say we just throw a session on that.us and chat.

**Comment by asharonbaltazar** [comment]
Sure, I'm free in about 15 mins. How does that sound?

**Comment by csell5** [comment]
well I'm streaming now. How about 3pm central? Not sure where you're located.

**Comment by asharonbaltazar** [comment]
I'm in central too. 3 pm sounds good.

**Comment by csell5** [comment]
> I'm in central too. 3 pm sounds good.

@asharonbaltazar Perfect. We can meet here: https://that.us/sessions/QzDQlvIeSIWj0okaNDMH

File: src/routes/Home.svelte [hunk 1]
```diff
@@ -44,7 +44,7 @@
   Newsletter?
 -->
 
-<div class="max-w-screen-xl mx-auto">
+<div class="relative max-w-screen-xl mx-auto z-10">
```
Comment (csell5) [inline src/routes/Home.svelte:47]: I know you made a comment about the zindex but I'm not sure how we'd get around that.

**Review by csell5 — CHANGES_REQUESTED** [review]

File: src/routes/Home.svelte (hunk 1 above)
Comment (asharonbaltazar) [reply src/routes/Home.svelte:47]: What should I do about this?

Comment (csell5) [reply src/routes/Home.svelte:47]: we talked about this yesterday.

**Comment by csell5** [comment]
Looks like you might still be working through this.. just re-request when you're ready.

**Comment by asharonbaltazar** [comment]
Well, other than your comment above and during a sight margin issue, I think it's ready. 

I do, however, want the menu to sort "stand out" from the rest of the page's white background. Any suggestions for this?

File: src/routes/Home.svelte (hunk 1 above)
Comment (asharonbaltazar) [reply src/routes/Home.svelte:47]: I don't think there's any way to get around this right now. Removing those classes causes the navbar to disappear

**Comment by csell5** [comment]
I still show the nav background not taking the whole screen 
image

**Comment by csell5** [comment]
I'm not sure I was totally clear in what I mean.. I was referring to the left side of the nav bar.

image

**Comment by asharonbaltazar** [comment]
Other than this, is there anything else that needs work?

**Comment by asharonbaltazar** [comment]
> I'm not sure I was totally clear in what I mean.. I was referring to the left side of the nav bar.

Addressed. Works fine on the emulator. Lemme know how it's for you.

**Comment by csell5** [comment]
I have to apologize, I don't seem to be expressing it correctly. Yes that does work, but we just needed the background to extend the whole way, not the contents of the nav bar. Now when you're on a bigger screen ( like me ) that nav is outside of the main content block.

image

So essentially the white background, should take the whole width. But the contents of the nav, should be visually centered like they were before.

**Comment by asharonbaltazar** [comment]
Can we just do a `flex justify-center` when on super large screens?

**Comment by asharonbaltazar** [comment]
I'm having trouble launching the project on the localhost. Unable to figure what's the issue.

**Comment by asharonbaltazar** [comment]
FIXED

**Comment by csell5** [comment]
Looking pretty hottt but it only lays out correctly on the home page. Check out https://thatus-kt2ybdv8u.vercel.app/members and https://thatus-kt2ybdv8u.vercel.app/partners.

This is in part as I should really have a dedicated layout for the "outside pages" vs the "inside pages" but if I remember correctly (??) those pages are pulling in the nav directly rather than a standard layout which has the nav.

**Comment by asharonbaltazar** [comment]
So if those are fixed, then what else is there to do?

**Comment by asharonbaltazar** [comment]
Okay, so these commits simply address the navbar. However, there are NEW issues (surprise, surprise). I think the Members page has a z-index problem. 

If you want, you can merge this, and start addressing those issues in separate places.

**Comment by csell5** [comment]
So I see. Rebasing it now. I'll review deeper here. I'm also working on a new `above the fold` layout that might work better for members and partners anyways.

**Review by csell5 — APPROVED** [review]

**Comment by csell5** [comment]
@asharonbaltazar nice job! thank you!

**Comment by asharonbaltazar** [comment]
I'll be eyeing the issues tab for more work. Thanks for the opportunity.

**Pull request merged**